HLTrigger/JetMET/src/HLTJetSortedVBFFilter.cc

```diff
@@ -28,59 +28,65 @@ using namespace std;
 //
 template<typename T>
 HLTJetSortedVBFFilter<T>::HLTJetSortedVBFFilter(const edm::ParameterSet& iConfig) : HLTFilter(iConfig)
- ,inputJets_   (iConfig.getParameter<edm::InputTag>("inputJets"   ))
- ,inputJetTags_(iConfig.getParameter<edm::InputTag>("inputJetTags"))
- ,mqq_         (iConfig.getParameter<double>       ("Mqq"         ))
- ,detaqq_      (iConfig.getParameter<double>       ("Detaqq"      ))
- ,detabb_      (iConfig.getParameter<double>       ("Detabb"      ))
- ,dphibb_      (iConfig.getParameter<double>       ("Dphibb"      )) 	
- ,ptsqq_       (iConfig.getParameter<double>       ("Ptsumqq"     ))
- ,ptsbb_       (iConfig.getParameter<double>       ("Ptsumbb"     ))
- ,seta_        (iConfig.getParameter<double>       ("Etaq1Etaq2"  ))
- ,value_       (iConfig.getParameter<std::string>  ("value"       ))
- ,triggerType_ (iConfig.getParameter<int>          ("triggerType" ))
+,inputJets_   (iConfig.getParameter<edm::InputTag>("inputJets"   ))
+,inputJetTags_(iConfig.getParameter<edm::InputTag>("inputJetTags"))
+,mqq_         (iConfig.getParameter<double>       ("Mqq"         ))
+,detaqq_      (iConfig.getParameter<double>       ("Detaqq"      ))
+,detabb_      (iConfig.getParameter<double>       ("Detabb"      ))
+,dphibb_      (iConfig.getParameter<double>       ("Dphibb"      ))
+,ptsqq_       (iConfig.getParameter<double>       ("Ptsumqq"     ))
+,ptsbb_       (iConfig.getParameter<double>       ("Ptsumbb"     ))
+,seta_        (iConfig.getParameter<double>       ("Etaq1Etaq2"  ))
+,njets_       (iConfig.getParameter<int>          ("njets"       ))
+,value_       (iConfig.getParameter<std::string>  ("value"       ))
+,triggerType_ (iConfig.getParameter<int>          ("triggerType" ))
 {
-  m_theJetsToken = consumes<std::vector<T>>(inputJets_);
-  m_theJetTagsToken = consumes<reco::JetTagCollection>(inputJetTags_);
+	m_theJetsToken = consumes<std::vector<T>>(inputJets_);
+	m_theJetTagsToken = consumes<reco::JetTagCollection>(inputJetTags_);
+	if(njets_<4) {
+		edm::LogWarning("LowNJets")<< "njets="<<njets_<<" it must be >=4. Forced njets=4.";
+		njets_=4;
+	}
 }
 
 
-template<typename T>
+	template<typename T>
 HLTJetSortedVBFFilter<T>::~HLTJetSortedVBFFilter()
 { }
 
 template<typename T>
 void
 HLTJetSortedVBFFilter<T>::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
-  edm::ParameterSetDescription desc;
-  makeHLTFilterDescription(desc);
-  desc.add<edm::InputTag>("inputJets",edm::InputTag("hltJetCollection"));
-  desc.add<edm::InputTag>("inputJetTags",edm::InputTag(""));
-  desc.add<double>("Mqq",200);
-  desc.add<double>("Detaqq",2.5);
-  desc.add<double>("Detabb",10.);
-  desc.add<double>("Dphibb",10.);
-  desc.add<double>("Ptsumqq",0.);
-  desc.add<double>("Ptsumbb",0.);
-  desc.add<double>("Etaq1Etaq2",40.);
-  desc.add<std::string>("value","second");
-  desc.add<int>("triggerType",trigger::TriggerJet);
-  descriptions.add(defaultModuleLabel<HLTJetSortedVBFFilter<T>>(), desc);
```

Comment:
I just add a "\t" instead of a space. 
Below you can see that I'm adding the same lines (plus "njets" description).